pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -114,7 +118,28 @@ func (r *InfraServiceAccounts) addServiceAccount(saName string, role authorizati
 		role.Annotations = map[string]string{}
 	}
 	role.Annotations[roleSystemOnly] = roleIsSystemOnly
-	r.saToRole[saName] = role
+
+	// TODO make this unnecessary
+	// we don't want to expose the resourcegroups externally because it makes it very difficult for customers to learn from
+	// our default roles and hard for them to reason about what power they are granting their users
+	for j := range role.Rules {
+		role.Rules[j].Resources = authorizationapi.NormalizeResources(role.Rules[j].Resources)
+	}
+
+	// TODO roundtrip roles to pick up defaulting for API groups.  Without this, the covers check in reconcile-cluster-roles will fail.
```

Comment:
They aren't separate here: https://github.com/openshift/origin/blob/master/pkg/authorization/api/v1/conversion.go#L95 

Also this is pre-existing.  Just moved it closer to its need.